_toc.yml

```diff
@@ -256,8 +256,6 @@ entries:
                 title: Schema registry
                 entries:
                   - file: docs/products/kafka/howto/enable-karapace
-                  - file: docs/products/kafka/howto/enable-schema-registry-authorization
-                  - file: docs/products/kafka/howto/manage-schema-registry-authorization
```

Comment:
Same comment, I believe we can't just get rid of the pages, we need to setup a redirect? Check the `_redirects` file